### DISCUSSION TITLE
How to disable benchmark - wazuh rules

### DISCUSSION THREAD

**Discussion by sidiyo** [wazuh/wazuh#16240] [Q&A]

**Comment by mdiego92** [comment]
Hello @sidiyo,

You can disable the **Security Configuration Assessment scans** from the manager and agents in its respective XML section by setting the `enabled` parameter to no:

```
  <sca>
    <enabled>no</enabled>
    <scan_on_start>yes</scan_on_start>
    <interval>12h</interval>
    <skip_nfs>yes</skip_nfs>
  </sca>
```

Articles for reference:

- https://documentation.wazuh.com/current/user-manual/capabilities/sec-config-assessment/how-to-configure.html
- https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/sca.html

If what you're looking for is to disable the scans for a group of agents, you can do it by setting up a centralized configuration as per our documentation: https://documentation.wazuh.com/current/user-manual/reference/centralized-configuration.html

Please let me know if you have any remaining questions.

Thank you very much!
